src/Main/NavBar.jsx

```diff
@@ -12,7 +12,8 @@ import {useStyles} from './styles';
 const google=window.google;
 
 function handleCredentialResponse(response){
-  axios.get('login',{withCredentials:false,'Authorization':response.credential})
+  console.log(response.credential)
```

Comment:
remove testing code please

Reply:
Yeah , I actually had removed test code in another commit after this.https://github.com/assembo/note-taker/pull/20/commits/4a4d65703703a6d513af3156e75c3ce3304f292c